src/client/src/orangehrmDashboardPlugin/components/MyActionSummeryView.vue

```diff
@@ -0,0 +1,197 @@
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <base-widget
+    icon="list-check"
+    :empty="isEmpty"
+    :empty-text="emptyText"
+    :loading="isLoading"
+    :title="$t('general.my_actions')"
+  >
+    <div v-if="myActions.length > 0" class="orangehrm-todo-list">
+      <div v-if="leaveRequestCount > 0" class="orangehrm-todo-list-item">
+        <oxd-icon-button
+          class="orangehrm-report-icon"
+          name="clock"
+          display-type="success"
+          @click="onClickPendingLeave"
+        />
+        <oxd-text tag="p" @click="onClickPendingLeave">
+          {{
+            $t('general.n_pending_leave_request', {
+              pendingActionsCount: leaveRequestCount,
+            })
+          }}
+        </oxd-text>
+      </div>
+      <div v-if="timeSheetCount > 0" class="orangehrm-todo-list-item">
+        <oxd-icon-button
+          class="orangehrm-report-icon"
+          name="calendar3"
+          display-type="warn"
+          @click="onClickPendingTimesheet"
+        />
+        <oxd-text tag="p" @click="onClickPendingTimesheet">
+          {{
+            $t('general.n_pending_time_sheet', {
+              pendingActionsCount: timeSheetCount,
+            })
+          }}
+        </oxd-text>
+      </div>
+      <div v-if="reviewCount > 0" class="orangehrm-todo-list-item">
+        <oxd-icon-button
+          class="orangehrm-report-icon"
+          name="person-fill"
+          display-type="danger"
+          @click="onClickPendingReview"
+        />
+        <oxd-text tag="p" @click="onClickPendingReview">
+          {{
+            $t('general.n_pending_performance_evaluate', {
+              pendingActionsCount: reviewCount,
+            })
+          }}
+        </oxd-text>
+      </div>
+      <div v-if="interviewCount > 0" class="orangehrm-todo-list-item">
+        <oxd-icon-button
+          class="orangehrm-report-icon"
+          name="people-fill"
+          display-type="info"
+          @click="onClickPendingInterview"
+        />
+        <oxd-text tag="p" @click="onClickPendingInterview">
+          {{
+            $t('general.n_pending_candidate_interview', {
+              pendingActionsCount: interviewCount,
+            })
+          }}
+        </oxd-text>
+      </div>
+    </div>
+  </base-widget>
+</template>
+<script>
+import {navigate} from '@ohrm/core/util/helper/navigation';
+import {APIService} from '@/core/util/services/api.service';
+import BaseWidget from '@/orangehrmDashboardPlugin/components/BaseWidget.vue';
+
+export default {
+  name: 'MyActionSummeryViewWidget',
```

Comment:
```suggestion
  name: 'MyActionSummaryWidget',
```
typo ~~Summery~~  ==> summary

Reply:
nice catch. @ManulMax please refactor rename the file name also